public/app/features/dashboard/containers/DashboardPage.tsx

```diff
@@ -333,10 +329,12 @@ export class UnthemedDashboardPage extends PureComponent<Props, State> {
           >
             <div className={styles.dashboardContent}>
               {initError && <DashboardFailed />}
-              {!editPanel && kioskMode === KioskMode.Off && (
-                <div aria-label={selectors.pages.Dashboard.SubMenu.submenu}>
-                  <SubMenu dashboard={dashboard} annotations={dashboard.annotations.list} links={dashboard.links} />
-                </div>
+              {!editPanel && (
+                <HidableInDisplayProfile pathInProfile="subMenu">
+                  <div aria-label={selectors.pages.Dashboard.SubMenu.submenu}>
```

Comment:
wonder when these can also be moved inside Submenu  (not related to this PR but these wrapper divs just look off :)